osquery/main/harnesses/CMakeLists.txt

```diff
@@ -5,6 +5,11 @@
 # the LICENSE file found in the root directory of this source tree.
 
 function(osqueryMainHarnesses)
+  if(OSQUERY_ENABLE_FUZZER_SANITIZERS AND NOT OSQUERY_BUILD_TESTS)
+    # Some fuzzers use include files from the tests
```

Comment:
Interesting, this should not be the case.

Reply:
It's because of sql_test_utils.h